10379.xml

```diff
@@ -0,0 +1,113 @@
+contributors may be used to endorse or promote products derived
+from this software without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+"AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+POSSIBILITY OF SUCH DAMAGE.
+-->
+
+<LWM2M xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://openmobilealliance.org/tech/profiles/LWM2M.xsd">
+    <Object ObjectType="MODefinition">
+        <Name>Measurement Metadata</Name>
+        <Description1>This object contains measurement metadata for sensor objects. 
+The corresponding sensor is identified with an object link.
+Measurement Quality Indicator and Measurement Quality Level values provided in this object have a precedence over 
+values provided in IPSO sensor object. If multiple object instances link to the same sensor object instance then 
+the metadata provided by the largest instance id takes precedence.</Description>
+    </Description1>
+    <ObjectID>10379</ObjectID>
+    <ObjectURN>urn:oma:lwm2m:x:10379</ObjectURN>
+    <LWM2MVersion>1.1</LWM2MVersion>
+    <ObjectVersion>1.0</ObjectVersion>
+    <MultipleInstances>Multiple</MultipleInstances>
+    <Mandatory>Optional</Mandatory>
+    <Resources>
+        <Item ID="0">
+            <Name>Linked Sensor</Name>
+            <Operations>R</Operations>
+            <MultipleInstances>Single</MultipleInstances>
+            <Mandatory>Mandatory</Mandatory>
+            <Type>Objlnk</Type>
+            <RangeEnumeration></RangeEnumeration>
+            <Units></Units>
+            <Description>Link to the sensor object instance that this metadata relates to.
+            </Item>
+            <Item ID="1">
+                <Name>Measurement Quality Reason Code</Name>
+                <Operations>R</Operations>
+                <MultipleInstances>Multiple</MultipleInstances>
+                <Mandatory>Optional</Mandatory>
+                <Type>Integer</Type>
+                <RangeEnumeration>0..1024</RangeEnumeration>
+                <Units></Units>
+                <Description>Measurement quality degradation reason reported by a smart sensor. 
+A degraded measurement has Measurement Quality Indicator value 1-3 or Measurement Quality Level < 100.
+0: UNKNOWN Reason is unkown, 1: UNDER_RANGE Measured value is under measurement range, 
```

Comment:
```suggestion
0: UNKNOWN Reason is unknown
1: UNDER_RANGE Measured value is under measurement range 
```

I'd recommend having each value in a new line (and without commas; perhaps full stop at the end like in the other resource)